docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -82,8 +85,10 @@ provided through our Web App or you can use our Python package and build a scrip
 
               Lightly S3 connection config
 
-      If you don`t know how to fill out the form, follow the full tutorial to
-      `create a Lightly dataset connected to your S3 bucket <https://docs.lightly.ai/getting_started/dataset_creation/dataset_creation_aws_bucket.html>`_.
+         If you don't know how to fill out the form, follow the full tutorial to create
+         a Lightly dataset connected to your bucket: :ref:`S3 <dataset-creation-aws-bucket>`, 
+         :ref:`GCS <dataset-creation-gcloud-bucket>`, 
+         :ref:`Azure <dataset-creation-azure-storage>`.
```

Comment:
Use the full names
 `AWS Secure Storage Solution (S3)`
- `Google Cloud Storage (GCS)`
- `Azure Blob Storage (Azure) `